RELEASING.md

```diff
@@ -1,7 +1,7 @@
 ## Release Checklist
 
 ### Prerelease (several days prior)
-- [Notify translators](https://www.transifex.com/projects/p/id-editor/announcements/) of the impending release
+- [Notify translators](https://app.transifex.com/openstreetmap/communication/?q=project%3Aid-editor) of the impending release
```

Comment:
I can access this URL but I do not know if an announcement can be created from here. I read the steps on https://help.transifex.com/en/articles/6224264-announcements-discussions-messages so the URL should be okay. Someone with access should test this

Note that I tested https://app.transifex.com/openstreetmap/id-editor/communication but this does not work

Reply:
The new link is perfect: as an admin one can create new announcements from there. Thanks :+1: